packages/api/models/base.js

```diff
@@ -281,6 +281,15 @@ class Manager {
     return this.dynamodbDocClient.batchWrite(params).promise();
   }
 
+  addTimeStampsToItem(item) {
```

Comment:
This logic is abstracted from the base so that provider, rule and collection can override it.

Reply:
Is this unnecessary given that the endpoints for provider/rule/collections were updated to explicitly set these properties? This is just ensuring that `updatedAt` is always set to now but that `createdAt` is overridden using the value on the passed-in object?

Reply:
Yes, in theory, because this default behavior overrides the updatedAt value set in the endpoint.